nifi-nar-bundles/nifi-prometheus-bundle/nifi-prometheus-processors/src/test/java/org/apache/nifi/processors/prometheus/PrometheusRemoteWriteBatchTest.java

```diff
@@ -0,0 +1,147 @@
+/*
```

Comment:
Even though GitHub Actions passed, these tests fail for me locally with a "No route to host" exception. The host/port should be fixed if a server is started for the test, or if a Prometheus instance is required to run the test, it should be an integration test instead (ending in `IT` rather than `Test`)